app/Http/Controllers/Api/Manager/AttendanceController.php

```diff
@@ -35,49 +35,32 @@ class AttendanceController extends Controller
      */
     public function store(StoreRequest $request): JsonResponse
     {
-        $managerId = $request->user()->id;
-
-        /** @var Instructor $manager */
-        $manager = Instructor::with('managings')->findOrFail($managerId);
-
-        $instructorIds = $manager->managings->pluck('id')->toArray();
-        $instructorIds[] = $manager->id;
-
-        $courseIds = Course::with('instructor')
-            ->whereIn('instructor_id', $instructorIds)
-            ->pluck('id')
-            ->toArray();
-
-        /** @var Course $course */
-        $course = Course::find($request->course_id);
+        /** @var Attendance $attendance */
```

Comment:
ここのphpdocの指定が間違っているので、修正をお願いします。
正しくは次のようになります。

```suggestion
        /** @var Course $course */
```